feg/radius/src/config/samples/radius.tataipx.config.json

```diff
@@ -23,9 +23,9 @@
                                 {
                                     "name": "akatataipx",
                                     "config": {
-                                        "IpxEndpoint": "https://carrieridpoc.mvoipctsi.com:36443/ss7gatewaysvcs",
-                                        "Username": "MapFBtclUser",
-                                        "Password": "X70:-1g$M-Q2^0"
```

Comment:
This will still be contained in git history - is someone taking care of chaging the credentials of this server/taking it offline?

Reply:
Yes, I'm working on it.